backends/README.md

```diff
@@ -32,13 +32,15 @@ So, although Netdata collects metrics every second, it can send to the backend s
     -   **prometheus** is described at [prometheus page](prometheus/) since it pulls data from Netdata.
 
     -   **prometheus remote write** (a binary snappy-compressed protocol buffer encoding over HTTP used by
-        **Elasticsearch**, **Gnocchi**, **Graphite**, **InfluxDB**, **Kafka**, **OpenTSDB**, **PostgreSQL/TimescaleDB**,
-        **Splunk**, **VictoriaMetrics**, and a lot of other [storage
+        **Elasticsearch**, **Gnocchi**, **Graphite**, **InfluxDB**, **Kafka**, **OpenTSDB**, **PostgreSQL**, **TimescaleDB**, **Splunk**, **VictoriaMetrics**, and a lot of other [storage
```

Comment:
Hi @joelhans ,

You also split PostgreSQL and TimescaleDB here, I think it is safe to return here.

Reply:
@thiagoftsm Reverted here as well.